### PR TITLE
🎨 clean up markdown issues

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

File: template.md [hunk 1]
```diff
@@ -103,7 +105,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
 ### Schedule constraints
 
-_Schedule constraints should be supplied here **48 hours** before the meeting begins so that the Chairs can take them into account when preparing the schedule._
+*Schedule constraints should be supplied here **48 hours** before the meeting begins so that the Chairs can take them into account when preparing the schedule.*
```
Comment (ljharb) [inline template.md:108]: why is use of the underscore an issue?

File: template.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="" />
+<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="Ecma logo" />
```
Comment (ljharb) [inline template.md:2]: might as well update the may agenda also

File: template.md (hunk 1 above)
Comment (ctcpip) [reply template.md:108]: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md049---emphasis-style-should-be-consistent

File: template.md (hunk 2 above)
Comment (ctcpip) [reply template.md:2]: ✅

File: template.md (hunk 1 above)
Comment (ljharb) [reply template.md:108]: not sure i buy the reasoning, but either way if we want to enforce markdownlint shouldn't we be doing so?

Comment (ctcpip) [reply template.md:108]: certainly could.  we did it in `how-we-work`, so why not?

File: template.md [hunk 3]
```diff
@@ -21,15 +20,17 @@ Temporal.ZonedDateTime.from('2021-10-25T10:00[Europe/London]')
...
 - Allen Wirfs-Brock's [paper on standards committee participation for new attendees](http://wirfs-brock.com/allen/files/papers/standpats-asianplop2016.pdf)
 - TC39's documentation on [How to participate in meetings](https://github.com/tc39/how-we-work/blob/HEAD/how-to-participate-in-meetings.md)
 
 ## Agenda topic rules
 
 Deadline for advancement eligibility: [**CALCULATE TIME HERE**](https://www.timeanddate.com/countdown/generic?p0=1440&iso=20000101T18&msg=TC39%20Submission%20deadline)
-  - <sub>Note: this time is selected to be precisely 10 days prior to the start of the meeting</sub>
 
-1. Proposals not looking to advance may be added at any time; if after the deadline, please always use a pull request so that members are notified of changes. Note: an unmerged PR counts as ”added” for the purposes of this requirement.
+- *Note: this time is selected to be precisely 10 days prior to the start of the meeting*
```
Comment (ljharb) [inline template.md:31]: this changes the intentionally small text to be big and italicized

**Review by ljharb — APPROVED** [review]

**Pull request merged**